packages/flutter/lib/src/gestures/resampler.dart

```diff
@@ -6,6 +6,9 @@ import 'dart:collection';
 
 import 'events.dart';
 
+/// A callback for pointer events.
+/// 
+/// See: [PointerEventResampler.sample] and [PointerEventResampler.stop]. 
```

Comment:
nit: here and everywhere else: Can you replace "see:" with "Used by" to match style?